packages/main/src/Input.ts

```diff
@@ -425,6 +439,16 @@ class Input extends UI5Element implements SuggestionComponent, IFormInputElement
 	@property({ type: Boolean })
 	showClearIcon!: boolean;
 
+	/**
+	 * Defines whether the suggestions picker is open.
+	 * Once the picker is open only the input field can close the picker internally.
+	 * @default false
+	 * @public
+	 * @since rc.2.0.0
```

Comment:
we did not have such notation so far - `rc.2.0.0`, 2.0.0-rc.3 is correct, but I think we set 2.0.0 everywhere.